apply_patches_ext.sh

```diff
@@ -60,7 +60,7 @@ fi
 cp $DEVDIR/kernel/realsense/d4xx.c $DEVDIR/$1/${D4XX_SRC_DST}/drivers/media/i2c/
 if [[ "$JETPACK_VERSION" == "6.0" ]]; then
     # jp6 overlay
-    cp $DEVDIR/hardware/realsense/tegra234-camera-d4xx-overlay.dts $DEVDIR/$1/hardware/nvidia/t23x/nv-public/overlay/
+    cp $DEVDIR/hardware/realsense/tegra234-camera-d4xx-overlay*.dts $DEVDIR/$1/hardware/nvidia/t23x/nv-public/overlay/
```

Comment:
Can you also check if "apply_patches.sh" need it?

Reply:
"apply_patches.sh" file is already handled in https://github.com/IntelRealSense/realsense_mipi_platform_driver/pull/223.